from_mwt_ds/DataScience/estimate.py

```diff
@@ -25,12 +25,12 @@ def estimate_bucket(bucket: pd.DataFrame, config: dict) -> dict:
             result[policy_name + "_" + estimator_name] = estimated_reward   
     return result
 
-def estimate(input_files, config):
+def estimate(input_files, config, output_path):
     number_of_events = config["aggregation"]['num_of_events']
     result = []
```

Comment:
result = [] should be moved after line 31, we want to collect the result from each file(01.csv, 02.csv etc) and dump each result to corresponding .aggregated file.